modules/bibdk_openuserstatus/bibdk_openuserstatus.module

```diff
@@ -69,8 +69,8 @@ function bibdk_openuserstatus_mypage_link() {
     'title' => t('Userstatus'),
     'description' => 'Status for user\'s loans and reservations',
     'page callback' => 'preview_openuserstatus',
-    'access callback' => TRUE,
-    'access arguments' => 'user_edit_access',
+    'access callback' => 'user_edit_access',
```

Comment:
Jeg er ikke sikker på hvad der egentlig foregår her - men hvorfor er det nu 'access callback' der nu skal have værdien 'user_edit_access'?

Reply:
Man kan kalde user_edit_access funktionen direkte og så levere den TRUE/FALSE.
Det som var der før leverede hele tiden TRUE.

Reply:
https://drupal.stackexchange.com/questions/5968/whats-the-difference-between-access-arguments-and-access-callback